.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## What does this PR do?
+
+## Are there points in the code the reviewer needs to double check?
+
+## Why was this PR needed?
+
+## Screenshots (if relevant)
+
+## Does this PR meet the acceptance criteria?
+
+     - [ ] [Documentation created/updated](https://www.neomutt.org/guide/)
```

Comment:
maybe telling here to modify `doc/manual.xml.head` could be a good idea? (up until we've got a document that says so in http://neomutt.org)